src/fshtypes/Configuration.ts

```diff
@@ -0,0 +1,168 @@
+  experimental?: boolean;
+  date?: string;
+
+  // The first publisher's name in the YAML file will be used as IG.publisher.  The contact details
+  // and/or additional publishers is translated into IG.contact values.
+  publisher?: string;
+
+  // The first contact entry contains additional publisher contact info (if applicable).
+  // Those who need more control or want to add additional details to the contact values can use
+  // contact directly and follow the format outlined in the ImplementationGuide resource and
+  // ContactDetail.
+  contact?: ContactDetail[];
+
+  description?: string;
+  useContext?: UsageContext[];
+  jurisdiction?: CodeableConcept[];
+  copyright?: string;
+
+  // SUSHI will use id as both id and packageId in the IG unless a specific packageId is specified
+  packageId?: string;
+
+  license?: string;
+  fhirVersion: string[];
+  dependencies?: ImplementationGuideDependsOn[];
+  global?: ImplementationGuideGlobal[];
+
+  // Groups can control certain aspects of the IG generation.  The IG documentation recommends that
+  // authors use the default groups that are provided by the templating framework, but if authors
+  // want to use their own instead, they can use the mechanism below.  This will create
+  // IG.definition.grouping entries and associate the individual resource entries with the
+  // corresponding groupIds.
+  groups?: ConfigurationGroup[];
+
+  // The resources property corresponds to IG.definition.resource. SUSHI can auto-generate all of
+  // the resource entries based on the FSH definitions and/or information in any user-provided
+  // JSON resource files. If the generated entries are not sufficient or complete, however, the
+  // author can add entries here. If the reference matches a generated entry, it will replace the
+  // generated entry. If it doesn't match any generated entries, it will be added to the generated
+  // entries. The format follows IG.definition.resource with the following differences:
+  // * additional "omit" property to omit a FSH-generated resource from the resource list.
+  // * groupingId can be used, but top-level groups syntax may be a better option (see below).
+  resources?: ConfigurationResource[];
+
+  // The pages property corresponds to IG.definition.page. SUSHI can auto-generate the page list,
+  // but if the author includes pages in this file, it is assumed that the author will fully manage
+  // the pages section and SUSHI will not generate any page entries. If title is not provided, then
+  // the title will be generated from the file name.  If a generation value is not provided, it
+  // will be inferred from the file name extension.
+  pages?: ImplementationGuideDefinitionPage[];
+
+  // The parameters property represents IG.definition.parameter. For a partial list of allowed
+  // parameters see: https://confluence.hl7.org/display/FHIR/Implementation+Guide+Parameters
+  parameters?: ImplementationGuideDefinitionParameter[];
```

Comment:
So this is where `copyrightYear` and `releaseLabel` go right?

Reply:
Yes.  Sorry, that's a bit less clear here without the YAMLConfiguration and importer stuff I held back from this PR.  But you've got it-- if the YAML has `copyrightYear`, `copyrightyear`, `releaseLabel`, or `releaselabel`, they'll get added as `copyrightyear` and/or `releaselabel` parameters (as appropriate).